components/docs-chef-io/content/automate/servicenow_integration_reference.md

```diff
@@ -145,6 +145,38 @@ The **Field Maps** establishes a relationship between a field in an import set t
 
 {{< figure src="/images/automate/snow_integration_transform_map_scripts1.png" alt="Transform Map Scripts">}}
 
+## Discovery
+
+[ServiceNow](https://www.servicenow.com/) Discovery feature finds applications and devices on your network, and then updates the Configuration Management Database (CMDB) with the information it finds. It discovers both physical and logical components, including virtual machines, servers, storage, databases, applications, and more.
+
+Refer [Discovery](https://docs.servicenow.com/bundle/paris-it-operations-management/page/product/discovery/reference/r-discovery.html) page for information on Discovery and its types.
+
+### Identification and Reconciliation
+
+Reconciliation is the synchronization of two or more matching database segments to ensure consistency across them.
+
+The Identification and Reconciliation rules maintain the integrity of the CMDB by managing duplicate CIs and controlling updates to CIs when multiple data sources are used to create and update CI records. These rules help prevent duplication of CI records, reconcile CI attributes, reclassify CIs, and allow authoritative data sources to update CI records in the CMDB.
```

Comment:
This paragraph (line 156) needs to be at the top of this section. It makes the subject of this section more clear.

Reply:
Actually, I want to say what is reconciliation first and then start explaining other things. Is that not fine Ian?